Changed array to access 3rd element, instead of 4th

DaveVoyles opened the pull request:
It looks as though the array was previously trying to access the 4th element:  arguments[3]

When I believe the purpose was to access the 3rd: arguments[2]

Unless I misunderstood?

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

DaveVoyles committed:
Changed array to access 3rd element, instead of 4th

It looks as though the array was previously trying to access the 4th element:  arguments[3]

When I believe the purpose was to access the 3rd: arguments[2]

Unless I misunderstood?
